common/browser.go

```diff
@@ -205,15 +205,15 @@ func (b *Browser) initEvents() error {
 
 	action := target.SetAutoAttach(true, true).WithFlatten(true)
 	if err := action.Do(cdp.WithExecutor(b.ctx, b.conn)); err != nil {
-		return fmt.Errorf("unable to execute %T: %w", action, err)
+		return fmt.Errorf("executing setAutoAttach: %w", err)
```

Comment:
Would "executing setAutoAttach" be meaningful to users? Could we say:

```suggestion
		return fmt.Errorf("auto-attaching to browser frames: %w", err)
```

Reply:
Most of the errors relating to CDP will be confusing to users, so I don't think "auto-attaching to browser frames" would be an improvement. I think what this actually does is [enable auto-attaching to related targets](https://chromedevtools.github.io/devtools-protocol/tot/Target/#method-setAutoAttach), whatever that means, but even that would be confusing.

Maybe we should translate all of these errors into a generic "initializing events" error, but we can discuss that in a new issue.

Reply:
But, in that code's context, it's auto-attaching to browser pages (_not frames, sorry_). It sounds better to me than "executing setAutoAttach" or "initializing setAutoAttach" event because users can understand that we're tracking the browser pages. Anyway, let's discuss it later then.